work/worker.go

```diff
@@ -829,6 +887,60 @@ func (env *Task) commitTransaction(tx *types.Transaction, bc BlockChain, rewardb
 	return nil, receipt.Logs
 }
 
+func (env *Task) commitBundleTransaction(bundle *builder.Bundle, bc BlockChain, rewardbase common.Address, vmConfig *vm.Config) (error, *types.Transaction, []*types.Log) {
+	lastSnapshot := env.state.Copy()
+	gasUsedSnapshot := env.header.GasUsed
+	txs := []*types.Transaction{}
+	receipts := []*types.Receipt{}
+	logs := []*types.Log{}
+
+	for _, txOrGen := range bundle.BundleTxs {
+		var tx *types.Transaction
+		if gen, ok := txOrGen.(builder.TxGenerator); ok {
+			var err error
+			tx, err = gen(env.state.GetNonce(rewardbase) + 1)
```

Comment:
Why is `env.state.GetNonce(rewardbase) + 1` used while `env.state.GetNonce(rewardbase)` is used in L756?

Reply:
It seems that L756 is wrong. `env.state.GetNonce(rewardbase) + 1` should be used. Thanks.

Reply:
It was fixed at 600f7e8.

Reply:
https://github.com/kaiachain/kaia/blob/47d8d543e61c7e8bbe23b731d0e5e32f72cc81d9/blockchain/tx_pool.go#L794-L796
Isn't `env.state.GetNonce(rewardbase)` correct? `getNonce(from) == tx.Nonce()` is allowed.